aea/__init__.py

```diff
@@ -21,6 +21,7 @@
 
 import inspect
 import os
+import aea.crypto  # triggers registry population
```

Comment:
this calls `aea.crypto.__init__.py`, which contains the instructions for the population of the registry

Reply:
Interestingly, this line caused the `setup.py build` to fail on CI.
The reason is that we import from `aea` to retrieve the extras, but that causes other third party packages to be imported, and those attempt fails because they are not installed yet.

Solved by removing the obsolete retrieval of the extras (see the changes in `setup.py`).